tokens/src/core/global/spacing.json

```diff
@@ -1,36 +1,89 @@
 {
+  "$type": "dimension",
   "spacing": {
     "spacer": {
-      "base": { "value": "1rem", "type": "dimension", "source": "$spacer", "description": "Basic space value" },
-      "0": { "value": "0", "type": "dimension", "description": "Space value of level 0" },
-      "1": { "value": "calc({spacing.spacer.base} * .25)", "type": "dimension", "description": "Space value of level 1" },
-      "1,5": { "value": "calc({spacing.spacer.base} * .375)", "type": "dimension", "description": "Space value of level 1.5" },
-      "2": { "value": "calc({spacing.spacer.base} * .5)", "type": "dimension", "description": "Space value of level 2" },
-      "2,5": { "value": "calc({spacing.spacer.base} * .75)", "type": "dimension", "description": "Space value of level 2.5" },
-      "3": { "value": "{spacing.spacer.base}", "type": "dimension", "description": "Space value of level 3" },
-      "3,5": { "value": "calc({spacing.spacer.base} * 1.25)", "type": "dimension", "description": "Space value of level 3.5" },
-      "4": { "value": "calc({spacing.spacer.base} * 1.5)", "type": "dimension", "description": "Space value of level 4" },
-      "4,5": { "value": "calc({spacing.spacer.base} * 2)", "type": "dimension", "description": "Space value of level 4.5" },
-      "5": { "value": "calc({spacing.spacer.base} * 3)", "type": "dimension", "description": "Space value of level 5" },
-      "5,5": { "value": "calc({spacing.spacer.base} * 4)", "type": "dimension", "description": "Space value of level 5.5" },
-      "6": { "value": "calc({spacing.spacer.base} * 5)", "type": "dimension", "description": "Space value of level 6" }
+      "0": {
+        "$value": "0",
+        "$description": "Space value of level 0"
+      },
+      "1": {
+        "$value": "calc({spacing.spacer.base} * .25)",
+        "$description": "Space value of level 1"
+      },
+      "2": {
+        "$value": "calc({spacing.spacer.base} * .5)",
+        "$description": "Space value of level 2"
+      },
+      "3": {
+        "$value": "{spacing.spacer.base}",
+        "$description": "Space value of level 3"
+      },
+      "4": {
+        "$value": "calc({spacing.spacer.base} * 1.5)",
+        "$description": "Space value of level 4"
+      },
+      "5": {
+        "$value": "calc({spacing.spacer.base} * 3)",
+        "$description": "Space value of level 5"
+      },
+      "6": {
+        "$value": "calc({spacing.spacer.base} * 5)",
+        "$description": "Space value of level 6"
+      },
+      "base": {
+        "source": "$spacer",
+        "$value": "1rem",
+        "$description": "Basic space value"
+      },
+      "1,5": {
```

Comment:
[clarification] Is the comma in `"1,5"` and other spacing tokens intended, or should it be a period (i.e., `"1.5"`)?

Reply:
It doesn't matter for the final result. But I agree with you, I replaced the comma with a period (by analogy to the value in the description)

[inform] Implemented in follow-up PR: https://github.com/openedx/paragon/pull/3207